index.html

```diff
@@ -6,11 +6,67 @@
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
+  <link rel="stylesheet" href="index.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <div class="container">
+    <header>
+      <nav>
+        <a href="services.html">Services</a> <a href="#">Product</a> <a href="#">Vision</a> <a href="#">Features</a> <a href="#">About</a> <a href="#">Contact</a>
+      </nav><img alt="Great Idea! Company logo."
+              class="logo"
+              src="img/logo.png">
+    </header>
```

Comment:
It's a good practice to format your HTML markup in a manner it would be easier to read. For e.g, each of the anchor tags should be on a single line. You could install `Prettier - Code formatter` Vscode plugin. When you install it, you can automatically format the document by pressing `SHIFT + option + F`